### DISCUSSION TITLE
能不能加入图片翻译功能啊

### DISCUSSION THREAD

**Discussion by srsman** [immersive-translate/immersive-translate#1430] [Ideas]
### 新功能 | New Feature

1.
2.
3.

### 为什么您想要添加这些功能 | Why you want to add these features

1.
2.
3.

**Comment by keyboard3** [comment]
描述下场景需求

**Reply by srsman** [reply]
1、某些图文混排的网页或插入图片的PDF，图片上的文字，保存为图片格式的表格之类
2、漫画

**Reply by keyboard3** [reply]
垂直类的漫画翻译靠谱一些。 通用图片翻译对推理要求更高。会优先考虑漫画翻译
